polling/periodic_sequence/workflows.py

```diff
@@ -38,6 +38,7 @@ async def run(self, name: str) -> str:
             except ActivityError:
                 workflow.logger.error("Activity failed, retrying in 1 seconds")
             await asyncio.sleep(1)
-            workflow.continue_as_new(name)
 
-        raise Exception("Polling failed after all attempts")
+        workflow.continue_as_new(name)
+        # unreachable
+        return ""
```

Comment:
```suggestion
```

`continue_as_new` return type hint has https://docs.python.org/3/library/typing.html#typing.NoReturn, so shouldn't need this part

Reply:
I've made the changes. Let's have the CI workflows check it.